src/types.ts

```diff
@@ -104,30 +117,28 @@ export interface IAddress {
   hasSearchedCoordinates: boolean
   id: number
   isHidden: boolean
-  latitude: number
-  longitude: number
   lotNum: number | null
   numberAndStreet: string
   state: string
   zipCode: string
 }
 
 export interface IMHBOListing {
-  address: IAddress
+  address: ILatLongAddress
   id: number
   entityType: IEntityType
   listingTypeId?: IListingTypeID
 }
 
 export interface IUnparsedMHBOListing {
-  address: IUnparsedAddress
+  address: IUnparsedLatLongAddress
   id: number
   entityType: string
   listingTypeId: string
-  url: string
 }
 
 export interface IMobileHome extends IMHBOListing {
+  address: IAddress
```

Comment:
Ah, cool. I didn't know whether IMobileHome with an address:IAddress attribute could be cast to a IMHBOListing with an address:ILatLongAddress and considered equivalent.